tests/serialize/prefixgen.ttl

```diff
@@ -1,18 +1,18 @@
 
 @prefix n1: <http://csarven.ca/#>.
-@prefix n0: <http://purl.org/dc/elements/1.1/>.
+@prefix dc: <http://purl.org/dc/elements/1.1/>.
```

Comment:
More a question than a suggestion, but should this be using the DC Terms namespace (http://purl.org/dc/terms/), as opposed to this DC Elements namespace? If you do just want the 15 DC Elements, then is the preferred prefix `dc11`, as suggested here: https://stackoverflow.com/questions/47519315/what-is-the-difference-between-dublin-core-terms-and-dublin-core-elements-vocabu.

That StackOverflow also quotes a recommendation to use the DC Terms namespace, referring to DC Elements as 'legacy', i.e., "**_Over time, however, implementers are encouraged to use the semantically more precise dcterms: properties, as they more fully follow emerging notions of best practice for machine-processable metadata._**".

Secondly (regardless of the first question), but since the Turtle serializations for both DC Terms and DC Elements use the prefix `dcterms` themselves, I'd suggest that that is their preferred prefix, and so would suggest using that prefix here too. (It sure would be nice if everyone actually stipulated a preference by using [vann:preferredNamespacePrefix](https://vocab.org/vann/#preferredNamespacePrefix), but what can you do!):
```suggestion
@prefix dcterms: <http://purl.org/dc/terms/>.
```

Reply:
I don't have a horse in that race.  I went to prefix.cc to get the prefixes listed there for the namespaces in these files, all of which I targeted because of their previous (well, current, until something gets merged) use of `@prefix XML: <http://www.w3.org/2001/XMLSchema#>` which just made me itch.

Maybe your much more detailed contemplation belongs in the issue (#472) rather than this PR, which I've noted only partly addresses the issue?

Reply:
No, I think the topic in both issues is pretty focused - i.e., the topic in this issue is purely Dublin Core, with the only questions being:
1. whether to reference the DCTerms or DCElements namespace? (My vote would be: `http://purl.org/dc/terms/`).
2. what prefix to use for 'DCStuff'? (My vote would be `dcterms:`).

I totally understand your using `prefix.cc` (I keep meaning to reach out to Richard again!), but it's really pretty much an ad-hoc, for-guidance-only tool. So, in the Dublin Core case at least, I think the references in the StackOverflow [link](https://stackoverflow.com/questions/47519315/what-is-the-difference-between-dublin-core-terms-and-dublin-core-elements-vocabu) I posted above would trump Prefix.cc.

I think the XML prefix issue is very focused too, so better addressed in #472 (although my vote would be `xsd` !).

Reply:
@pmcb55 - This (@474) is a Pull Request.  The other (#472) is an Issue, which is (partially) fixed by this Pull Request.  As I created both #472 and #474, I think I can say whether either is "purely" focused on Dublin Core (neither is) or "very focused" on the XML prefix (that's what first caught my eye and brought me to create Issue #472, but it's not all I focused on in either Issue #472 or Pull Request #474).

Reply:
Ah, right-you-are @TallTed - sorry for any confusion :( .
Anyway, I've thrown in my 2cents worth - I was only trying to offer you guys my thoughts, specifically as I've asked myself **_exactly_** these very same questions in my own work over the years. Cheers.